Wiki/zh-Tw.js

```diff
@@ -519,6 +521,48 @@ exports.MODULES = {
       SUBTITLE: "設定舉報頻道",
       SUGGESTION: "您設定的頻道需要是不公開且伺服器管理員可以看到的"
     }
+  },
+  ANTI_EXTERNAL_LINKS: {
```

Comment:
A translation from Traditional Chinese is missing here.

P.S.: I have added some translations that have been recycled to build the documentation for this module.